src/code/z_play.c

```diff
@@ -186,9 +186,9 @@ void Play_Destroy(GameState* thisx) {
     EffectSs_ClearAll(this);
     CollisionCheck_DestroyContext(this, &this->colChkCtx);
 
-    if (gTrnsnUnkState == 3) {
-        TransitionUnk_Destroy(&sTrnsnUnk);
-        gTrnsnUnkState = 0;
+    if (gTransitionTileState == TRANS_TILE_PROCESS) {
+        TransitionTile_Destroy(&sTransitionTile);
+        gTransitionTileState = TRANS_TILE_OFF;
     }
```

Comment:
```suggestion
    if (gTransitionTileState == TRANS_TILE_DONE) {
        TransitionTile_Destroy(&sTransitionTile);
        gTransitionTileState = TRANS_TILE_OFF;
    }
```
I think there was a comment to sync this more with:
```c
typedef enum {
    /* 0 */ PAUSE_BG_PRERENDER_OFF, // Inactive, do nothing.
    /* 1 */ PAUSE_BG_PRERENDER_SETUP, // The current frame is only drawn for the purpose of serving as the pause background.
    /* 2 */ PAUSE_BG_PRERENDER_PROCESS, // The previous frame was PAUSE_BG_PRERENDER_SETUP, now apply prerender filters.
    /* 3 */ PAUSE_BG_PRERENDER_DONE, // The pause background is ready to be used.
    /* 4 */ PAUSE_BG_PRERENDER_MAX
} PauseBgPreRenderState;
```
Is there part of this that doesn't line up with this?

Reply:
(my comment didnt secify that it *should* be synced with pause bg, I thought it already *was*)
But, if it can be synced, then sure

Reply:
I could probably sync them up, the main hesitation for me is that for the transition state 3, it continually calls update so doesn't seem `done` to me. However, I do see that the same thing happen for `PAUSE_BG_PRERENDER` where 3 continually calls `PreRender_RestoreFramebuffer`.

Reply:
Question. Thoughts on changing `Done` to `Ready`? indicating the prerender or transition are ready to be used?

Reply:
I like "ready"